.github/workflows/ci.yaml

```diff
@@ -8,8 +8,13 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.8", "3.9", "3.10"]
+        os: ["windows-latest", "ubuntu-latest", "macos-latest"]
+        python-version: ["3.9", "3.10", "3.11"]
+        experimental: [false]
+        include:
+          - python-version: "3.11"
+            os: "ubuntu-latest"
+            experimental: true
```

Comment:
There's no point in having this if you don't implement the `experimental` install step to install unstable dependencies.

Reply:
Ok, let me try, though this is outside my comfort zone. I was looking at how you for Satpy, perhaps this will work...

Reply:
Oh you know what, I think the current install in Satpy is broken with some recent changes in pandas (their build depends on a package not available from the package source we have). Let's see how it goes.

Reply:
Oh and the caching will complicate this. Let me see if I can help...